### ISSUE TITLE
[BugReport] There is an incorrect layout in multi select mode.

### ISSUE THREAD

**Issue by simjs1984** [ParkSangGwon/TedBottomPicker#37]
There is an incorrect layout in multi-select mode.
I saw incorrect layout while using this library.
sometimes layout vibrates in my project.
so, I tried to run your sample project.
and, I faced the same issue on my devices that Galexy7edge and Redmi note4.
please check attached video.
[bug-reporting.mp4.zip](https://github.com/ParkSangGwon/TedBottomPicker/files/862087/bug-reporting.mp4.zip)

**Comment by ParkSangGwon** [comment]
@simjs1984 Thanks about your video report.
Can you show me your code?
I want to know how to you launch `TedBottomPicker`

**Comment by simjs1984** [comment]
It's my code.

```java
TedBottomPicker pickerFragment = new TedBottomPicker.Builder(Activity.this)
                .setOnMultiImageSelectedListener(urls -> replyInputView.setAttachedImageUris(urls);)
                .setPreviewMaxCount(100)
                .create();
        pickerFragment.show(getSupportFragmentManager());
```
and I saw the same problem in your sample project.
the video was recorded by the sample project.

thanks

**Comment by ParkSangGwon** [comment]
@simjs1984 Yes, I saw this issue.
But this issue is not always shown.
I will check this issue and test.
Thanks about your report

**Comment by ParkSangGwon** [comment]
@simjs1984 Now this issue solved at v1.0.12
